app/routes/receive_package.js

```diff
@@ -14,8 +13,20 @@ export default AuthorizeRoute.extend({
     controller.set("autoGenerateInventory", true);
     controller.set("inputInventory", false);
     model.get("inventoryNumber");
+    if (model.get("isReceived")) {
+      window.history.back();
+    }
     if (!model.get("inventoryNumber")) {
       controller.generateInventoryNumber();
     }
+  },
+
+  resetController(controller, isExiting, transition) {
+    if (
+      controller.get("package.item") &&
+      transition.targetName === "offer.search_label"
```

Comment:
`nit`: Conditions can be specified here in reverse order. The easily evaluated one can be added at first which will come out of condition-check if it fails early.